os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -66,6 +66,16 @@ static int ns_flash_write(uint32_t address, uint32_t len, uint8_t *data)
 	return ret;
 }
 
+int rtl_ss_flash_write(uint32_t address, uint32_t len, uint8_t *data)
```

Comment:
Let's split commits with single topic. This commit has too many different topics.

Reply:
I splited it to secure_storage test commit. Thank you.

Reply:
1. This commit has changes on os and apps together. Let's split them as different commit.
2. Is this necessary in the se module? Is this a security operation?